app/scripts/services/nodeService.js

```diff
@@ -1,9 +1,12 @@
 'use strict';
 
-DrupalIssuesApp.factory('nodeService', ['$http', '$q', 'nodeEndpoint', function($http, $q, nodeEndpoint) {
+DrupalIssuesApp.factory('nodeService', ['$http', '$q', 'nodeEndpoint', 'userEndpoint', function($http, $q, nodeEndpoint, userEndpoint) {
   return {
     getNode: function(nid) {
       return $http({method: 'get', url: nodeEndpoint + nid});
+    },
+    getUser: function(uid) {
+      return $http({method: 'get', url: userEndpoint + uid + '/feed'});
```

Comment:
Just discovered that you don't need a uid and a regular username works!

Testing:
* Meself: https://www.drupal.org/project/issues/user/mglaman/feed
* User w/ space: view-source:https://www.drupal.org/project/issues/user/Damien%20Tournoud/feed

I think the UX is better to go by username, especially since user profiles no longer have the uid in the URL.